pepper-apis/src/main/resources/db-changes/schema/DDP-4439-language-popup-settings.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="charlotte" id="20200828-language-popup-setting">
+        <addColumn tableName="study_settings">
+            <column name="should_display_language_change_popup" type="boolean" defaultValueBoolean="false">
+                <constraints nullable="false"/>
+            </column>
+        </addColumn>
+    </changeSet>
+
+    <changeSet author="charlotte" id="20200831-skip-language-popup-profile">
+        <addColumn tableName="user_profile">
+            <column name="should_skip_language_popup" type="boolean">
+                <constraints nullable="true"/>
+            </column>
+        </addColumn>
+    </changeSet>
+
+    <changeSet author="charlotte" id="20201002-rename-skip-language-popup-profile">
```

Comment:
Since this hasn't gone to `dev` or any of the environments, I think it will be cleaner to edit the original changeset instead.